src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -0,0 +1,160 @@
+<template>
+    <header class="slds-modal__header slds-clearfix">
+        <h2 class="slds-text-heading_medium slds-text-heading--medium slds-hyphenate header-string slds-p-top_large slds-p-horizontal_medium">
+            {header}
+        </h2>
+    </header>
+
+    <lightning-record-edit-form record-id={recordId}
+                                object-api-name="npe03__Recurring_Donation__c"
+                                onsubmit={handleSubmit}
+                                onerror={handleError}
+                                onsuccess={handleSuccess}>
+
+        <div class="slds-modal__content slds-p-top_none slds-p-horizontal_medium slds-p-bottom_medium">
+            <template if:true={isLoading}>
+                <div class="slds-grid slds-grid_align-center">
+                    <div class="slds-is-relative slds-size_1-of-1">
+                        <lightning-spinner alternative-text="Loading"></lightning-spinner>
+                        <c-util-stencil row-count="20" column-count="2"></c-util-stencil>
+                    </div>
+                </div>
+            </template>
+            <template if:true={isRecordReady}>
+
+                <div class={cssEditForm}>
+                    <template if:true={hasError}> 
+                        <c-util-page-level-message title={errorMessage.header}
+                                                subtitle={errorMessage.detail}
+                                                subtitle-class-override="slds-p-horizontal_medium slds-p-vertical_xx-small"
+                                                variant="error">
+                        </c-util-page-level-message>
+                    </template>
+                
+                    <div class="slds-section slds-is-open">
+                        <h3 class="slds-section__title slds-theme_shade">
+                            <span class="slds-truncate slds-p-horizontal_small" title="Section Title">{customLabels.donorSectionHeader}</span>
+                        </h3>
+                        <div aria-hidden="false" class="slds-section__content">
+                            <lightning-layout horizontal-align="spread" multiple-rows="true">
+                                <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6">
+                                    <lightning-input-field field-name={fields.contact.apiName} value={contactId}>
```

Comment:
When a Contact lookup is searched for contacts, the popup dialog is displayed across the whole page. Also, a new Contact cannot be created as it could be on a standard Salesforce edit page layout. Is this expected behaviour?
<img width="982" alt="Screen Shot 2020-05-22 at 11 16 38 AM" src="https://user-images.githubusercontent.com/28743286/82698885-7f7ce700-9c20-11ea-8025-f2adf940ad5e.png">

<img width="1464" alt="Screen Shot 2020-05-22 at 11 16 50 AM" src="https://user-images.githubusercontent.com/28743286/82698890-84419b00-9c20-11ea-964d-19a7cbab39f2.png">

Reply:
The + New Record button will be display over here
![Screen Shot 2020-05-22 at 12 37 19 PM](https://user-images.githubusercontent.com/57271120/82703585-08981c00-9c29-11ea-8078-61b150340b32.png)